README.md

```diff
@@ -18,36 +18,33 @@ The Sunshine Conversations API offers multiple [versions](https://docs.smooch.io
 
 ### Maven users
 
-Add the following remote repo to your project's POM:
-
-```xml
-<project ...>
-<repositories>
-    <repository>
-      <id>zendesk-sunshine-conversations</id>
-      <url>https://smoochorg.bintray.com/maven/</url>
-    </repository>
- </repositories>
-</project>
-```
-
 Add this dependency to your project's POM:
 
 ```xml
 <dependency>
     <groupId>com.zendesk</groupId>
     <artifactId>sunshine-conversations-client</artifactId>
-    <version>9.4.4</version>
+    <version>9.4.3</version>
     <scope>compile</scope>
 </dependency>
 ```
 
 ### Gradle users
 
-Add this dependency to your project's build file:
+Add the following remote repo to your project's POM:
+
+```groovy
+repositories {
+  mavenCentral()
+}
+```
+
+Gradle >v7 should already have it listed by default.
+
+Next, add this dependency to your project's build file:
```

Comment:
do we have to specify "under repositories" or anything here?

Reply:
There's a `dependencies` class in the `build.gradle` file which looks like this:

```java

dependencies {
    testCompile group: 'junit', name: 'junit', version: '4.12'
    compile "io.smooch:api:2.0.0"
    compile 'io.jsonwebtoken:jjwt-api:0.11.2'
    runtime 'io.jsonwebtoken:jjwt-impl:0.11.2',
            // Uncomment the next line if you want to use RSASSA-PSS (PS256, PS384, PS512) algorithms:
            //'org.bouncycastle:bcprov-jdk15on:1.60',
            'io.jsonwebtoken:jjwt-jackson:0.11.2' // or 'io.jsonwebtoken:jjwt-gson:0.11.2' for gson
}
```

I just made teh assumption that most people would figure out to put it in the block.

Reply:
I don't think including this snippet would hurt (better to be too specific?) wdyt?